src/tests/problem-9.test.js

```diff
@@ -0,0 +1,5 @@
+const sum = require('../problem-9');
+
+test('Rerunt the sum of array', () => {
```

Comment:
Typo ;) I personally use a spell checker in my IDE 😅  Check it out https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker